packages/e2e-tests/specs/editor/plugins/inner-blocks-prioritized-inserter-blocks.test.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	createNewPost,
+	deactivatePlugin,
+	getAllBlockInserterItemTitles,
+	insertBlock,
+	closeGlobalBlockInserter,
+} from '@wordpress/e2e-test-utils';
+
+const QUICK_INSERTER_RESULTS_SELECTOR =
+	'.block-editor-inserter__quick-inserter-results';
+
+describe( 'Prioritized Inserter Blocks Setting on InnerBlocks', () => {
+	beforeAll( async () => {
+		await activatePlugin(
+			'gutenberg-test-innerblocks-prioritized-inserter-blocks'
+		);
+	} );
+
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	afterAll( async () => {
+		await deactivatePlugin(
+			'gutenberg-test-innerblocks-prioritized-inserter-blocks'
+		);
+	} );
+
+	describe( 'Quick inserter', () => {
+		it( 'uses defaulting ordering if prioritzed blocks setting was not set', async () => {
+			const parentBlockSelector =
+				'[data-type="test/prioritized-inserter-blocks-unset"]';
+			await insertBlock( 'Prioritized Inserter Blocks Unset' );
+			await closeGlobalBlockInserter();
+
+			await page.waitForSelector( parentBlockSelector );
+
+			await page.click(
+				`${ parentBlockSelector } .block-list-appender .block-editor-inserter__toggle`
+			);
+
+			await page.waitForSelector( QUICK_INSERTER_RESULTS_SELECTOR );
+
+			await expect( await getAllBlockInserterItemTitles() ).toHaveLength(
```

Comment:
I noticed this test in the follow up PR and it doesn't seem to add any value. We should either test something about the existence/non-existence of some blocks or remove it.

Reply:
Yes I agree. Seems pointless. It was probably me copying boilerplate code about.

We should probably change it to test what happens if you pass an empty value or empty array as the prioritized blocks argument.